CMakeLists.txt

```diff
@@ -3,14 +3,8 @@ project(libkeyfinder)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 
-option(BUILD_STATIC_LIBS "Build library statically" OFF)
-if(BUILD_STATIC_LIBS)
-  set(LIBMODE "STATIC")
-else()
-  set(LIBMODE "SHARED")
-endif()
-
-add_library(keyfinder "${LIBMODE}"
+option(BUILD_SHARED_LIBS "Build dynamic library" OFF)
```

Comment:
default to ON? Or default to ON for Windows?

Reply:
On Windows it can't be `ON` afaik. The dll is invalid because there are no dllexports I think.

Reply:
We should either enforce a static build on Windows or add support for dllexports.

Reply:
Done in https://github.com/mixxxdj/libKeyFinder/pull/2/commits/9f824f03201da4e264c14b504321f348b822e4ae and https://github.com/mixxxdj/libKeyFinder/pull/2/commits/2cf9f21819b10dad7f897b93196675a50bcdce65.